translib/app_utils.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/Azure/sonic-mgmt-common/translib/db"
 	"github.com/Azure/sonic-mgmt-common/translib/ocbinds"
 	"github.com/Azure/sonic-mgmt-common/translib/tlerr"
+	"google.golang.org/grpc/codes"
```

Comment:
Can you run 'go mod tidy' to update go.mod?

Reply:
Done, thanks Kwan